board/imgtec/pistachio_bub/fdt.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2016 Imagination Technologies
+ * Author: Francois Berder <francois.berder@imgtec.com>
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include <common.h>
+
+#if defined(CONFIG_OF_LIBFDT) && defined(CONFIG_OF_BOARD_SETUP)
+
+#include <winbond-otp.h>
+#include "otp.h"
+
+#define WIFI_STA_MAC_ADDRESS_OFFSET	0x1003
+#define WIFI_AP_MAC_ADDRESS_OFFSET	0x1009
+#define DCXO_OFFSET			0x2003
+
+DECLARE_GLOBAL_DATA_PTR;
+
+int __ft_board_setup(void *blob, bd_t *bd)
+{
+	ft_cpu_setup(blob, bd);
+
+	return 0;
+}
+int ft_board_setup(void *blob, bd_t *bd)
+	__attribute__((weak, alias("__ft_board_setup")));
+
+static void fixup_wifi(void *blob)
+{
+	int node, len;
+	u_char wifi_sta_mac_addr[MAC_ADDR_LEN], wifi_ap_mac_addr[MAC_ADDR_LEN];
+	char dcxo;
+	char *rf_params_prop;
+
+	if (check_otp_version(VERSION_REG0_OFFSET)
+	||  check_otp_version(VERSION_REG1_OFFSET))
+		return;
+
+	node = fdt_path_offset(blob, "/uccp@18480000");
```

Comment:
should we add an alias for Wi-Fi node as we do for ethernet, as change in dts node name will require a change in the code ?